utils/less.xml

```diff
@@ -10,7 +10,7 @@
   <homepage>http://gnuwin32.sourceforge.net/packages/less.htm</homepage>
   <needs-terminal/>
   <implementation arch="Windows-i486" id="sha1new=b5ab96a0747d4c6e6cbe2610cf7b0a97225a7d65" license="GPL v2 (GNU General Public License); BSD License (revised)" released="2006-01-03" version="394-3">
-    <requires interface="https://apps.0install.net/lib/pcre.xml">
+    <requires interface="http://repo.roscidus.com/lib/pcre" version="7.0">
```

Comment:
This should probably be `<requires interface="https://apps.0install.net/lib/pcre.xml" version="7.0">`?

Reply:
Yes it should
fixed